src/xpk/commands/workload.py

```diff
@@ -447,6 +451,13 @@ def workload_create(args) -> None:
   else:
     xpk_print('No gcsfuse Storages to add detected')
 
+  if len(gcpfilestore_storages) > 0:
+    xpk_print(
+        f'Detected gcp filestores instances to add: {gcpfilestore_storages}'
+    )
+  else:
+    xpk_print('No detected gcp filestore instances to add detected.')
```

Comment:
'No detected gcp filestore instances to add detected.'
 ->
 'No gcp filestore instances to add detected.'